src/components/SearchInput/elements.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const Container = styled(({...props}) => <div {...props} />)`
+
+`
+
+export const SearchSubmit = styled(({...props}) => <button {...props} />)`
+
+`
+
+export const SearchInput = styled(({...props}) => <input placeholder="Search" {...props} />)`
```

Comment:
Extract input into its own base Component, then import it here and add additional stylings if necessary.